scripts/automator/packages/manager-apt

```diff
@@ -1,2 +1,2 @@
 # Package repo: https://packages.ubuntu.com/
-packages+=(ccache xvfb libtool build-essential libsdl2-dev libsdl2-net-dev libncurses-dev libopusfile-dev)
+packages+=(ccache libtool build-essential autotools-dev automake libsdl2-dev libsdl2-net-dev libncurses-dev libopusfile-dev)
```

Comment:
Annoyingly, `apt` on (at least these tested ARM platforms) does not provide a complete auto-stack with `build-essentials`, so this covers these cases.